### PR TITLE
doc: add rc.2 announcement blog

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Add RC.2 announcement blog based on the Release Notes at https://github.com/notaryproject/roadmap/pull/87.

See the [preview](https://deploy-preview-141--notarydev.netlify.app/blog/2023/announcing-notation-rc2/) page.

### CODE REVIEW

File: content/en/blog/2023/announcing-notation-rc2.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+```
+$ notation verify --user-metadata io.wabbit-networks.buildId=123 sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+Successfully verified signature for sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+The artifact is signed with the following user metadata.
+
+KEY                         VALUE
+io.wabbit-networks.buildId  123
+```
+
+In addition, storing signatures in the registry using OCI image manifest was added into RC.3, you can use a flag `--signature-format image` to explicitly specify uploading image manifest to a registry when signing an artifact.
```
Comment (iamsamirzon) [inline content/en/blog/2023/announcing-notation-rc2.md:57]: RC.3 ??

Comment (iamsamirzon) [reply content/en/blog/2023/announcing-notation-rc2.md:57]: Also, we should give a reason/benefit of using "image manifest" method. Perhaps we can say something like, for registries that do not yet support OCI 1.1 spec, or for compatibility with existing tools.

File: content/en/blog/2023/announcing-notation-rc2.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Last but not least, `--debug` and `--verbose` flags were added to all CLI commands, providing debug and troubleshooting capability for Notation CLI.
+
+### Ecosystem integration
+
+[Ratify](https://github.com/deislabs/ratify) is a verification engine for Kubernetes which enables verification of artifact security metadata and admits for deployment only those that comply with policies you create. It integrated Notation-go v1.0.0-rc.2 which allows you to verify a container image signed by Notation.
```
Comment (iamsamirzon) [inline content/en/blog/2023/announcing-notation-rc2.md:63]: Do we need to say which release of Ratify integrated Notation RC-2 ?

**Review by iamsamirzon — COMMENTED** [review]
Overall looks good, have provided inline comments

File: content/en/blog/2023/announcing-notation-rc2.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+### Enhanced stability with more testing coverage
+
+As the second RC release for v1.0.0, we aim to deliver a production-ready security product.  Comparing with RC.1, the E2E testing framework built on [ginkgo](https://onsi.github.io/ginkgo/) was set up in RC.2 and major test cases were added to Notation CLI. Meanwhile, unit test coverage was also increased by [5.92%](https://app.codecov.io/gh/notaryproject/notation-go?search=&trend=3%20months). It definitely enhanced the program robustness and project stability.
+
+## Credits
+
+We would like to specially thank the Notation maintainers, contributors, and the broader Notation community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release. Sending credits to the following contributors who made great contributions to RC.2.
+
+@toddysm
```
Comment (toddysm) [inline content/en/blog/2023/announcing-notation-rc2.md:77]: I would put those in alphabetical order. I am not sure my name should be first in that list from contribution point of view ;) and this gives wrong impression. Alphabetical order is at least impartial.

**Review by toddysm — CHANGES_REQUESTED** [review]

File: content/en/blog/2023/announcing-notation-rc2.md (hunk 3 above)
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc2.md:77]: Good catch, I will adjust this contributor list in alphabetical order

Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc2.md]: Updated to use alphabetical order

File: content/en/blog/2023/announcing-notation-rc2.md (hunk 2 above)
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc2.md]: Added the detailed version of Ratify

**Review by toddysm — CHANGES_REQUESTED** [review]

File: content/en/blog/2023/announcing-notation-rc2.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+author:  "Notation Release Team"
+date: 2023-02-27
+draft: false
+---
+
+The Notary v2 project maintainers are pleased to announce the release of Notation v1.0.0-RC.2, including Notation CLI, Notation-go, and Notation-core-go library. This is the first Notation release of 2023. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds enhancements including inspecting signatures associated with signed artifacts, storing signatures in the registry using OCI image manifest, adding user-defined metadata to signature payload, and more!
```
Comment (toddysm) [inline content/en/blog/2023/announcing-notation-rc2.md:12]: Would be nice to list all significant changes as a bulleted list.

**Review by toddysm — CHANGES_REQUESTED** [review]

File: content/en/blog/2023/announcing-notation-rc2.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+```
+$ notation verify --user-metadata io.wabbit-networks.buildId=123 sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+Successfully verified signature for sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+The artifact is signed with the following user metadata.
+
+KEY                         VALUE
+io.wabbit-networks.buildId  123
+```
+
+In addition, storing signatures in the registry using OCI image manifest was added into RC.2, you can use a flag `--signature-format image` to explicitly specify uploading image manifest to a registry when signing an artifact. This brings backward compatibility which enables Notation work with the registries that do not support OCI Spec v1.1 yet.
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc2.md]: My typing issue. I have fixed it.

File: content/en/blog/2023/announcing-notation-rc2.md (hunk 4 above)
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc2.md]: Good suggestion! I changed it to a bulleted list

**Review by toddysm — APPROVED** [review]
LGTM

File: content/en/blog/2023/announcing-notation-rc2.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+```
+$ notation verify --user-metadata io.wabbit-networks.buildId=123 sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+Successfully verified signature for sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+The artifact is signed with the following user metadata.
+
+KEY                         VALUE
+io.wabbit-networks.buildId  123
+```
+
+In addition, storing signatures in the registry using OCI image manifest was added into RC.2, you can use a flag `--signature-manifest image` to explicitly specify uploading image manifest to a registry when signing an artifact. This brings backward compatibility which enables Notation work with the registries that do not support OCI Spec v1.1 yet.
```
Comment (iamsamirzon) [inline content/en/blog/2023/announcing-notation-rc2.md:62]: Wondering if we should remove this line, or rephrase it.  In RC-3 we are making the default to be image-spec and even in  OCI 1.1 registry the default will be image-manifest.

**Review by iamsamirzon — APPROVED** [review]

File: content/en/blog/2023/announcing-notation-rc2.md (hunk 6 above)
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc2.md:62]: It should be clarified in RC.3's release notes or blog.

**Review by yizha1 — APPROVED** [review]
LGTM

**Pull request merged**